script.js

```diff
@@ -1,6 +1,8 @@
 ﻿const wrapper = document.createElement("div");

```

Comment:
It would be a good idea to wrap all this code in class or in function to exclude it from global scope